addons/calendar/models/calendar_attendee.py

```diff
@@ -96,14 +96,21 @@ def _send_invitation_emails(self):
         """ Hook to be able to override the invitation email sending process.
          Notably inside appointment to use a different mail template from the appointment type. """
         self._send_mail_to_attendees(
-            self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
+            self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False),
+            force_send_wlimit=True,
         )
 
-    def _send_mail_to_attendees(self, mail_template, force_send=False):
+    def _send_mail_to_attendees(self, mail_template, force_send=False, force_send_wlimit=False):
         """ Send mail for event invitation to event attendees.
             :param mail_template: a mail.template record
             :param force_send: if set to True, the mail(s) will be sent immediately (instead of the next queue processing)
+            :param Union[int, bool] force_send_wlimit: force_send up to a number of attendee, default value if True
```

Comment:
Indeed, my only concern would be 100 seems like it might be a bit high for something that happens in the background. It makes more sense when you're explicitly trying to send a mail vs just moving stuff around in the UI. But I guess 100 attendees is rare enough.

Reply:
Also we should check that 'send_mail_to_attendees" is not called in a loop using force_send=True (some calls are updated to be done in a single big recordset instead of per event basis; did not check everything was updated or under control).